allure-web/src/components/tree/TreeView.js

```diff
@@ -34,10 +37,10 @@ class TreeView extends View {
         this.$(e.currentTarget).parent().toggleClass('node__expanded');
     }
 
-    @on('click .statuses-filter')
+    @on('click .tree__control_statuses')
```

Comment:
according to BEM, you need to use `-` in element names: `.tree__control-statuses`
Then active will be `tree__control-statuses_active`. Easier to understand, what is the element name, what is the modification